android/app/src/main/java/com/mapeo/FlagSecureModule.java

```diff
@@ -0,0 +1,60 @@
+package com.mapeo;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import android.app.Activity;
+import android.view.WindowManager;
+
+public class FlagSecureModule extends ReactContextBaseJavaModule 
+{
```

Comment:
nit: update formatting of `{ }` based on common java conventions (see AppInfo module as an example)

(i'm using https://marketplace.visualstudio.com/items?itemName=redhat.java in VSCode to do formatting on save if that's helpful)